### ISSUE TITLE
Expansion Sets, or Minimizing Graph Expansion Transitions

### ISSUE THREAD

**Issue by everbeek** [thechiselgroup/biomixer#354]
This certainly does look like a duplicate of #387. The unique idea is defaulting expansions to hide previous node sets...which I do not like. This idea is related to the re-focusing idea; this issue itself is no longer useful though.

Following from #318 in general, and dependent on #353 for clean usage.

Consider this interaction style:
User expands mappings for a particular concept. Currently, those nodes are added to the current view. I cannot think of a good reason to look at those nodes _and_ the previous set of nodes. Suppose when certain expansions happen, the rest of the graph is minimized, hidden, or abstracted into a stand-in node.

In case the user _does_ want to see it all, they should be able to get the hidden nodes back easily.
